INSTALL.rst.txt

```diff
@@ -27,16 +31,16 @@ PREREQUISITES
 
 SciPy requires the following software installed for your platform:
 
-1) Python__ 2.6.x or newer
+1) Python__ 2.7 or >= 3.4
 
 __ http://www.python.org
 
-2) NumPy__ 1.6.2 or newer (note: SciPy trunk at times requires latest NumPy
+2) NumPy__ 1.7.1 or newer (note: SciPy trunk at times requires latest NumPy
    trunk).
```

Comment:
This is simply not true anymore, so I'll just change the sentence to ``NumPy >= 1.7.1``